6_4.py

```diff
@@ -0,0 +1,14 @@
+import cv2
+
+def find_code(path_image):

```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/